imagersite/imager_images/views.py

```diff
@@ -1,16 +1,189 @@
-from django.shortcuts import render
+from django.shortcuts import render, get_object_or_404
 from imager_images.models import Photo, Album
+from django.contrib.auth.decorators import login_required
+from django.http import HttpResponseRedirect, HttpResponse
+from django.core.exceptions import PermissionDenied
+from django.conf import settings
+from .forms import *
+from djgeojson.serializers import Serializer as GeoJSONSerializer
 
 
+import Algorithmia
+import base64
+
+
+def get_faces(path):
+    with open(settings.MEDIA_ROOT + "/" + path, "rb") as img:
+        bimage = base64.b64encode(img.read())
+    Algorithmia.apiKey = "Simple simivSeptsC+ZsLks5ia0wXmFbC1"
+    result = Algorithmia.algo('/ANaimi/FaceDetection').pipe(bimage)
+    faces = []
+    for rect in result:
+        face = Face()
+        face.name = "Petter Rabbit"
+        face.x = rect['x']
+        face.y = rect['y']
+        face.width = rect['width']
+        face.height = rect['height']
+        faces.append(face)
+        for face in faces:
+            face.save()
```

Comment:
You are saving each face in `faces` each time you add a new face to `faces`.  Fix please.